base_riskscore/verification/verification_code/code/get_cv_sl_seeds.R

```diff
@@ -0,0 +1,11 @@
+#-----------------------------------------------
+# obligatory to append to the top of each script
+renv::activate(project = here::here("..", "..", ".."))
+source(here::here("..", "..", "..", "_common.R"))
+#-----------------------------------------------
+
```

Comment:
We should remove this file and any references to the output it creates, since only a single random seed is now used.